bidict/_bidict.py

```diff
@@ -73,10 +79,10 @@ def clear(self):
         self._inv.clear()
 
     def pop(self, key, *args):
-        """Like :py:meth:`dict.pop`, keeping bidirectionality intact."""
-        ln = len(args) + 1
-        if ln > 2:
-            raise TypeError('pop expected at most 2 arguments, got %d' % ln)
+        """Like :py:meth:`dict.pop`, maintaining bidirectionality."""
+        l = len(args) + 1
```

Comment:
(My rationale was that the reasons not to are more important when the variable is used in a lot of places that make it hard to read it -- in this case its use seems limited and obvious enough that no matter what font is used, it's clear)